### PR TITLE
(Routing-mechanism): set up the routing in the App

### PR DESCRIPTION
**Pull request by ravinalamada**
Hi team,

Nivia team has finished the Routing in this branch. Please have a look at it.

Thanks in advance,

Refs: #45e4gg-T1

### CODE REVIEW

File: src/Pages/HomePage.tsx [hunk 1]
```diff
@@ -17,7 +19,10 @@ export const HomePage: React.FC<Props> = ({ footerLinks }) => {
         <HeaderContainer>
           <Header item={loggedIn} />
         </HeaderContainer>
-        <RoleSelectOnboard />
+        <Switch>
+          <Route exact path='/' component={Test}></Route>
+          <Route path='/registered' component={RoleSelectOnboard}></Route>
```
Comment (andrzej-onja) [inline src/Pages/HomePage.tsx]: create enum with route names and path string values

File: package.json [hunk 2]
```diff
@@ -14,13 +14,19 @@
     "@types/react-router-dom": "^5.1.7",
     "@types/styled-components": "^5.1.10",
     "axios": "^0.21.1",
+    "connected-react-router": "^6.9.1",
     "cors": "^2.8.5",
     "cypress": "7.6.0",
+    "cypress-react-router": "^1.0.2",
+    "history": "4.10.1",
+    "i": "^0.3.6",
+    "npm": "^7.20.2",
```
Comment (andrzej-onja) [inline package.json]: npm ?

File: package.json [hunk 3]
```diff
@@ -14,13 +14,19 @@
     "@types/react-router-dom": "^5.1.7",
     "@types/styled-components": "^5.1.10",
     "axios": "^0.21.1",
+    "connected-react-router": "^6.9.1",
     "cors": "^2.8.5",
     "cypress": "7.6.0",
+    "cypress-react-router": "^1.0.2",
+    "history": "4.10.1",
+    "i": "^0.3.6",
```
Comment (andrzej-onja) [inline package.json]: i?

File: package.json (hunk 2 above)
Comment (andrzej-onja) [reply package.json]: those are not necessary I guess

**Review by andrzej-onja — APPROVED** [review]

**Review by vakodrazan — APPROVED** [review]

**Review by tsipoy — APPROVED** [review]

**Pull request merged**